packages/core/src/utils/JsonTransformer.ts

```diff
@@ -1,15 +1,53 @@
+import type { Validate } from 'class-validator'
+
 import { instanceToPlain, plainToInstance, instanceToInstance } from 'class-transformer'
 
+import { ClassValidationError } from '../error/ClassValidationError'
+import { isValidationErrorArray } from '../error/ValidationErrorUtils'
+
+import { MessageValidator } from './MessageValidator'
+
+interface Validate {
+  validate?: boolean
+}
+
 export class JsonTransformer {
   public static toJSON<T>(classInstance: T) {
     return instanceToPlain(classInstance, {
       exposeDefaultValues: true,
     })
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  public static fromJSON<T>(json: any, Class: { new (...args: any[]): T }): T {
-    return plainToInstance(Class, json, { exposeDefaultValues: true })
+  public static fromJSON<T>(
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    json: any,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    Class: { new (...args: any[]): T },
+    options: Validate = { validate: true }
```

Comment:
This will mean the default value is not set when you call it with an object (e.g. `fromJson({ })` will mean validate is undefined which means false.)

We can always set a default (even if an empty object is provided) using the following syntax:
```suggestion
    { validate = true }: Validate = { }
```

--- 
or we can update it like this


```suggestion
    options?: Validate
```

so we can use it like this in the function body:
```
const validate = options?.validate ?? true
```